astroid/nodes/scoped_nodes/mixin.py

```diff
@@ -170,3 +173,19 @@ class ComprehensionScope(LocalsDictNodeNG):
     """Scoping for different types of comprehensions."""
 
     scope_lookup = LocalsDictNodeNG._scope_lookup
+
+    generators: List["nodes.Comprehension"]
+    """The generators that are looped through."""
+
+    def qname(self) -> str:
+        """Get the 'qualified' name of the node."""
+        return self.pytype()
```

Comment:
@cdce8p do you have thoughts about this? In the past I believe you were a vote against, see https://github.com/PyCQA/astroid/pull/1284#pullrequestreview-823449216.